subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/java/tasks/CreateDependencyInfoFileTask.java

```diff
@@ -52,4 +58,31 @@ public void setOutputFile(File outputFile) {
         this.outputFile = outputFile;
     }
 
+    /**
+     * See {@link Project#getGroup()}
+     */
+    public String getProjectGroup() {
```

Comment:
Can you go through all the public methods and the Javadoc on the class type and ensure it has the right level of detail? For example: how does the task use the 'group',  'version', and 'configuration' properties? The class javadoc should explain the details like the distinction between internal module dependencies and external dependencies and how we use version.